assets/app/styles/_log.less

```diff
@@ -109,6 +109,9 @@
     color: #ededed;
   }
 }
+.log-line-number:before {
```

Comment:
Wrote some code that generates log output and performance seems fine, even in Safari where we had the most problems. I can't really tell a difference even at 50K+ lines.